tests/unit/interpreter_v2.cpp

```diff
@@ -0,0 +1,1636 @@
+// Copyright 2022 Memgraph Ltd.
```

Comment:
When and how do you plan to migrate these changes to query-v2/storage-v3? I know we talked about it, but forgot the conclusion.

Reply:
In the PR after we merge this one